### PR TITLE
Adding labels for wallets, contrats and blockchain

### LINKED ISSUE #7: Add a const label to all metrics indicating the current blockchain.

### LINKED ISSUE #8: Add config file, and aliases for tracked contracts
Add an alias to contract address I want to listen. For example, when adding `0x128310273012ueaoidu`, add a string alias like `pepito`, that will be shipped as a const label.

As per consensus, in this task a config file will also be added, to use instead of the CLI arguments. Much cleaner!

**Pending**
- [x] Add a config file instead of CLI arguments https://github.com/thepalbi/ethereum-prometheus-exporter/pull/13
- [x] Add aliases for ERC20 tracked contracts

### PR DESCRIPTION
**Pull request by leandro-deveikis**

### CODE REVIEW

File: internal/collectors/contracts/erc20/transfer_event.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ type contractInfo struct {
 	Address  string
 	Symbol   string
 	Decimals uint8
+	Label    string
```
Comment (thepalbi) [inline internal/collectors/contracts/erc20/transfer_event.go]: I'd prefer this to be called `Name`

File: internal/collectors/contracts/erc20/transfer_event.go [hunk 2]
```diff
@@ -38,7 +39,7 @@ type ERC20TransferEvent struct {
 	bnGetter         BlockNumberGetter
 }
 
-func getContractInfo(contractAddr common.Address, contractClient bind.ContractCaller) (*contractInfo, error) {
+func getContractInfo(contractAddr common.Address, contractClient bind.ContractCaller, label string) (*contractInfo, error) {
```
Comment (thepalbi) [inline internal/collectors/contracts/erc20/transfer_event.go]: Same here, the argument `label` called `name`

File: internal/collectors/contracts/erc20/transfer_event.go [hunk 3]
```diff
@@ -55,22 +56,24 @@ func getContractInfo(contractAddr common.Address, contractClient bind.ContractCa
...
 			return nil, errors.Wrap(err, "failed to create ERC20 transfer evt collector")
 		}
-		info, err := getContractInfo(contractAddress, client)
+		info, err := getContractInfo(address, client, contractAddress.Name)
 		if err != nil {
 			return nil, err
 		}
 
-		log.Printf("Got info for %s, symbol %s\n", info.Address, info.Symbol)
+		log.Printf("Got info for %s, symbol %s, address_name %s\n", info.Address, info.Symbol, info.Label)
```
Comment (thepalbi) [inline internal/collectors/contracts/erc20/transfer_event.go]:
```suggestion
		log.Printf("Got info for %s, symbol %s, with name %s\n", info.Address, info.Symbol, info.Label)
```

File: internal/collectors/contracts/erc20/transfer_event.go [hunk 4]
```diff
@@ -79,7 +82,7 @@ func NewERC20TransferEvent(client ContractClient, contractAddresses []common.Add
 		desc: prometheus.NewDesc(
 			"erc20_transfer_event",
 			"ERC20 Transfer events count",
-			[]string{"contract", "symbol"},
+			[]string{"contract", "symbol", "address_name"},
```
Comment (thepalbi) [inline internal/collectors/contracts/erc20/transfer_event.go]:
```suggestion
			[]string{"contract", "symbol", "name"},
```

File: internal/collectors/eth/block_number.go [hunk 5]
```diff
@@ -11,14 +11,14 @@ type EthBlockNumber struct {
...
-func NewEthBlockNumber(rpc *rpc.Client) *EthBlockNumber {
+func NewEthBlockNumber(rpc *rpc.Client, label string) *EthBlockNumber {
 	return &EthBlockNumber{
 		rpc: rpc,
 		desc: prometheus.NewDesc(
 			"eth_block_number",
 			"number of the most recent block",
 			nil,
-			nil,
+			map[string]string{"blockchain_name": label},
```
Comment (thepalbi) [inline internal/collectors/eth/block_number.go]: In all cases, why don't we just call this label `blockchain`?

File: internal/collectors/contracts/erc20/transfer_event.go [hunk 6]
```diff
@@ -2,16 +2,16 @@ package erc20
...
 	"github.com/prometheus/client_golang/prometheus"
+	"github.com/thepalbi/ethereum-prometheus-exporter/internal/config"
 	"github.com/thepalbi/ethereum-prometheus-exporter/token"
+	"log"
+	"math"
+	"math/big"
+	"sync"
 )
 
 type BlockNumberGetter interface {
```
Comment (santihernandezc) [inline internal/collectors/contracts/erc20/transfer_event.go:17]: Why are we creating an `interface` with no functionality, just native-type fields? A `struct` seems more appropiate.

**Review by thepalbi — APPROVED** [review]

**Review by santihernandezc — CHANGES_REQUESTED** [review]

File: internal/collectors/contracts/erc20/transfer_event.go (hunk 6 above)
Comment (leandro-deveikis) [inline internal/collectors/contracts/erc20/transfer_event.go:17]: @santihernandezc the interface has only one method, I think you are looking it wrong

Comment (leandro-deveikis) [reply internal/collectors/contracts/erc20/transfer_event.go:17]: also, that was part of the original project, I dont think we modified it

Comment (leandro-deveikis) [reply internal/collectors/contracts/erc20/transfer_event.go:17]: `type BlockNumberGetter interface {
	BlockNumber(ctx context.Context) (uint64, error)
}`

**Pull request merged**